example_site/src/components/ConfigurationControls.js

```diff
@@ -257,9 +260,9 @@ export default function ConfigurationControls({ isOpen }) {
             />
           </div>
 
-          <div>
-            <h2>Editor</h2>
-            <h4>Editor Orientation: </h4>
+          <div className="configuration-controls-subsection">
+          <div className="setting-title">Editor</div>
```

Comment:
It looks like the indentation here is off - the `setting-title` div needs to be inside of `configuration-controls-subsection`, not at the same level